packages/vm/src/backend.rs

```diff
@@ -168,6 +175,8 @@ pub enum BackendError {
     // This is the only error case of BackendError that is reported back to the contract.
     #[error("User error during call into backend: {msg}")]
     UserErr { msg: String },
+    #[error("Crypto error: {msg}")]
+    CryptoErr { msg: String },
```

Comment:
`BackendError` should not have a `CryptoErr`, because we never call the backend. The terms are used as follows:

contract (cosmwasm-std) -> VM (cosmwasm-vm) -> VM's "backend" (blockchain via wasmvm)

But since the crypto APIs are implemented in the VM directly (in contrast to humanize/canonicalize which are chain specific), no backend is needed.

Reply:
I see. I had it the other way around :-) will fix this asap.